test/core/end2end/tests/connectivity.cc

```diff
@@ -65,14 +66,18 @@ static void test_connectivity(grpc_end2end_test_config config) {
   grpc_core::CqVerifier cqv(f.cq);
   child_events ce;
 
-  grpc_channel_args client_args;
-  grpc_arg arg_array[1];
+  grpc_arg arg_array[] = {
+      grpc_channel_arg_integer_create(
+          const_cast<char*>(GRPC_ARG_INITIAL_RECONNECT_BACKOFF_MS), 1000),
+      grpc_channel_arg_integer_create(
+          const_cast<char*>(GRPC_ARG_MAX_RECONNECT_BACKOFF_MS), 1000),
+      grpc_channel_arg_integer_create(
+          const_cast<char*>(GRPC_ARG_MIN_RECONNECT_BACKOFF_MS), 5000)};
   arg_array[0].type = GRPC_ARG_INTEGER;
```

Comment:
This overwrites the first value in the fixed list.

Reply:
fixed